crates/swc/tests/tsc.rs

```diff
@@ -20,6 +20,10 @@ use testing::{NormalizedOutput, Tester};
         "propertyAccessChain\\.3.ts",
         "objectRestNegative.ts",
         "objectRestPropertyMustBeLast.ts",
+        // shall panic because not finding private field
```

Comment:
I think we should not panic.
Instead of panic, we can point the usage of `private`s using `swc_common::error::HANDLER`.

Reply:
I think panic is fine for now.
Which do you prefer? Merge it as is, or improve the message?

Reply:
I'm not quite sure how to use HANDLER, could you take a look at my new commit?

Reply:
Using it is a bit hard as you should not use `panic`.
You have to return some dummy value like `None` and stop by using e.g. `return`

Reply:
Ok, let's do it later then